packages/wheels_driver_dbv2/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package>
+  <name>wheels_driver_dbv2</name>
+  <version>1.0.0</version>
+  <description>A interface between our WheelsCmd messagese and the adafruit driver</description>
+
+   <author email="info@duckietown.org">Mack</author>
+  
+  <maintainer email="mack@duckietown.org">Mack</maintainer>
+  <license>GPLv3</license>
+
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>duckietown_msgs</build_depend>
+  <build_depend>rospy</build_depend>
+  <build_depend>adafruit_drivers</build_depend>
+
+  <run_depend>duckietown_msgs</run_depend>
+  <run_depend>rospy</run_depend>
+  <run_depend>adafruit_drivers</run_depend>
```

Comment:
for shorter files, I think just "< depend >BALBLA</ depend>" is enough (it covers both the build and the run depends, and those should be used only when only one is needed)